crates/tako/src/scheduler/state.rs

```diff
@@ -248,6 +269,8 @@ impl SchedulerState {
 
         let ready_tasks = core.take_ready_to_assign();
         if !ready_tasks.is_empty() {
+            // Find worker with minimum of tasks but at least one, otherwise just return any worker
```

Comment:
I'm not sure that I understand this comment.

Also, can we move these hardcoded constants (like `1000`) into something like `const MIN_READY_TASKS_TO_STICK = ...` and put it on top of the scheduler? So that it easier to see all the "knobs".